enterprise-suite/scripts/build-operator.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+console_version="1.2.0"
+docker_registry="lightbend-docker-commercial-registry.bintray.io"
+image_name="enterprise-suite/console-operator"
+full_docker_name="${docker_registry}/${image_name}:${console_version}"
+
+operator-sdk new console-operator --type=helm --kind=LBConsole \
+    --api-version=lightbend.com/v1alpha1 --helm-chart-repo=https://repo.lightbend.com/helm-charts \
```

Comment:
Don't we need to coordinate the api version across lightbend? We should probably use a subdomain as well to distinguish Console CRDs from other projects, unless we want to centralize - maybe a question for the platform team.

Reply:
Good point.  Will ponder...

Reply:
For now, I added `api_version` as a variable at the top of the script.